### PR TITLE
Feat: component speaker v2

### LINKED ISSUE #79: Component: Speaker
The name is mandatory.

All the other informations are optionnal.

Image

Image

### PR DESCRIPTION
**Pull request by IounOm**
## What does it do?
- edit component speaker in PR #98 
- split component speaker to 2 component
  - SpeakerDetail -> for render single speaker
  - SpeakerGroup -> for render multiple speaker

## Screenshot
Screenshot 2567-10-09 at 18 22 14
Screenshot 2567-10-09 at 18 23 01

## Related issue
- close #79

### CODE REVIEW

**Review by JeanneGrenet — CHANGES_REQUESTED** [review]

File: src/components/speakerDetail.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+const SpeakerGroup = (
+  props: Readonly<{
+    speakers: CollectionEntry<"people">[];
+    className?: string;
+  }>,
+) => {
+  return (
+    <div className={cn("grid grid-cols-2 gap-4 lg:gap-8", props.className)}>
+      {props.speakers.map((speaker) => (
+        <SpeakerDetail speaker={speaker.data} />
```
[the remark this answers is not in the document]
Comment (IounOm) [reply src/components/speakerDetail.tsx]: fixed!

**Comment by IounOm** [comment]
Update 16 Oct 24
- add SpeakerCarousel component

Screenshot 2567-10-16 at 18 01 40
Screenshot 2567-10-16 at 18 02 00

**Comment by IounOm** [comment]
Update 16 Oct 24
- add SpeakerCarousel component

Screenshot 2567-10-16 at 18 07 10
Screenshot 2567-10-16 at 18 05 50

**Review by JeanneGrenet — APPROVED** [review]

File: src/components/speakerDetail.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+              </li>
+            ))}
+          </ul>
+        )}
+      </div>
+    </div>
+  );
+};
+
+const SpeakerGroup = (
```
Comment (houssembaltii) [inline src/components/speakerDetail.tsx:63]:
```suggestion
const Speakers = (
```

**Review by houssembaltii — APPROVED** [review]

File: src/components/speakerDetail.tsx (hunk 2 above)
Comment (IounOm) [reply src/components/speakerDetail.tsx]: fixed!

**Pull request merged**